src/services/api.js

```diff
@@ -0,0 +1,7 @@
+import axios from 'axios';
+
+const ROOT_URL = process.env.REACT_APP_BASE_URL;
+const signupUrl = `${ROOT_URL}/users/`;
+const signUpApi = (data) => axios.post(signupUrl, data);
+
+export default signUpApi;
```

Comment:
Please refactor this file since all our urls will be here. Refactor it so that  we export one object with all the axios functions

Reply:
Hello @matthenge ,
I have fixed that. Thanks